src/main/java/io/dockstore/provision/DOSPluginUtil.java

```diff
@@ -99,9 +98,7 @@ HttpURLConnection createConnection(ImmutableTriple<String, String, String> immut
             return validConn;
 
         } catch ( IOException e) {
-            System.err.println("createConnection() Error: " + e.getMessage());
-            e.printStackTrace();
-
+            System.err.println("Error: " + e.getMessage() + ". Returning null");
```

Comment:
I don't think you should say `Returning null` (missed this last time). This is a message an end user will see, and it seems like a debugging message.  Please push back if you disagree.

Reply:
No, I agree with you. Will remove